webrecorder/browsertrix
# "Add new" dialog for all resources

Resolves https://github.com/webrecorder/browsertrix-cloud/issues/1197



### Changes
- Replaces individual "New" buttons in home page with dropdown button in header
- Refactors required step of new workflow and new collection into dialog

### Manual testing

1. Log in. Verify "Add New..." dropdown button is in home dashboard header
2. Click "Add New..." > "Crawl Workflow" > Crawl type. Verify you're taken to the correct form.
3. Click "Start Over". Verify crawl type dialog is shown
4. Go to Crawling page and click "New Workflow". Verify dialog is shown as expected
5. Go to home page
6. Click "Add New..." > "Collection". Verify dialog is shown
7. Enter name and click "Create Collection". Verify you're taken to the crawl selection form.
8. Go to Collections page and click "New Collection". Verify dialog is shown as expected

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| Home | <img width="1132" alt="Screenshot 2023-09-25 at 2 26 15 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/257ca4f0-f8ac-4dde-92ce-de718bef652d"> |
| Home | <img width="191" alt="Screenshot 2023-09-25 at 2 26 19 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/2781134a-6157-4e4a-93ec-be13d7baad75"> |
| Home/Crawling - New Workflow dialog | <img width="765" alt="Screenshot 2023-09-25 at 2 26 26 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/628ada46-14a1-4771-a57f-314f0c60231f"> |
| Home/Collections - New Collection dialog | <img width="752" alt="Screenshot 2023-09-25 at 2 44 37 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/f3549bd6-edef-4cbe-b065-893342d4a50b"> |
